.travis.yml

```diff
@@ -7,7 +7,7 @@ jdk: oraclejdk8
 before_install:
   - sudo apt-get update -q
   - sudo apt-get install -y curl unzip
-  - pip install --user codecov
+  - travis_wait 20 pip install --user codecov
```

Comment:
wut, do we actually spend time on this?

Reply:
![rtfm](https://user-images.githubusercontent.com/7294857/29858991-bbbb7fde-8d68-11e7-9f95-5658ad96e2f6.png)